templates/core/terraform/state-store/output.tf

```diff
@@ -4,4 +4,8 @@ output "endpoint" {
 
 output "cosmosdb_account_name" {
   value = azurerm_cosmosdb_account.tre-db-account.name
-}
\ No newline at end of file
+}
+
+output "id" {
```

Comment:
Nit picking: This could've been aligned with the "cosmosdb_account_name" either by adding "cosmosdb_account_" prefix or removing the prefix from the name output. Something to consider, but not worth blocking this PR over.

Reply:
I agree, and thought about doing that. However didn't have much time to consider the implications and fix elsewhere, so left it as.